apps/web/src/pages/templates/hooks/usePreviewPushTemplate.ts

```diff
@@ -45,13 +42,15 @@ export const usePreviewPushTemplate = (locale?: string) => {
   });
 
   useEffect(() => {
-    getPushPreview({
-      locale,
-      content: previewData.current.templateContent,
-      payload: previewData.current.processedVariables,
-      title: previewData.current.templateTitle,
-    });
-  }, [getPushPreview, locale, previewData]);
+    if (!disabled) {
+      getPushPreview({
+        locale,
+        content: templateContent,
+        payload: JSON.parse(processedVariables),
```

Comment:
Sorry, I meant that here it should stay the same (`PayloadType.payload`  is type string)
```suggestion
        payload: processedVariables,
```
But in the implementation it should parse it  
<img width="793" alt="Screen Shot 2024-02-15 at 14 17 43" src="https://github.com/novuhq/novu/assets/7778680/cc8ff2f5-a461-4ba0-a331-8604851c5f62">

Reply:
Oh ok, my bad.
Updating it